### PR TITLE
feat: add mattermost chat

### PR DESCRIPTION
**Pull request by linuxbandit**

### CODE REVIEW

**Review by serge1peshcoff — APPROVED** [review]

File: .env.example [hunk 1]
```diff
@@ -40,6 +40,8 @@ ELASTIC_PASSWORD=
 
 SLACK_WEBHOOK_GRAFANA=
 
+MM_USERNAME=
```
Comment (serge1peshcoff) [inline .env.example:43]: can we rename it to MATTERMOST_USERNAME? it can seem unobvious what it does

**Comment by linuxbandit** [comment]
we will not use mattermost

**Comment by WikiRik** [comment]
@linuxbandit why not? Because we are using Slack more now?

**Comment by linuxbandit** [comment]
correct, plus it still takes effort to administrate a self-hosted solution

**Comment by serge1peshcoff** [comment]
but we'll still hit 10k messages limit right?

**Comment by linuxbandit** [comment]
slack is for communication, not for archiviation

**Pull request closed without merging**